modules/cluster-node-tuning-operator-verify-profiles.adoc

```diff
@@ -28,3 +28,19 @@ worker-b         openshift-node            True      False      6h28m
 * `APPLIED`: `True` if the TuneD daemon applied the desired profile. (`True/False/Unknown`).
 * `DEGRADED`: `True` if any errors were reported during application of the TuneD profile (`True/False/Unknown`).
 * `AGE`: Time elapsed since the creation of Profile object.
+
+The ClusterOperator/node-tuning object also contains useful information about the Operator and its node agents' health. For example, Operator misconfiguration will be reported via ClusterOperator/node-tuning status messages.
+
```

Comment:
Needs an introductory sentence before the code block, such as "To get status information about the `ClusterOperator/node-tuning` object, run the following command:"